dataplane/standalone/cpusink/sink/cpusink.go

```diff
@@ -34,37 +29,26 @@ import (
 	fwdpb "github.com/openconfig/lemming/proto/forwarding"
 )
 
-// TODO: A better way to get this value.
 const (
 	contextID  = "lucius"
-	addr       = "10.0.2.2:50000"
-	ip2meTable = "ip2me"
+	IP2MeTable = "ip2me"
 )
 
-//export StartSink
-func StartSink() {
-	conn, err := grpc.Dial(addr, grpc.WithTransportCredentials(insecure.NewCredentials()))
-	if err != nil {
-		log.Fatal(err)
-	}
-	sink := &cpuSink{
-		client:       fwdpb.NewForwardingClient(conn),
-		ethDevToPort: make(map[string]string),
-	}
-
-	go func() {
-		if err := sink.receivePackets(context.Background()); err != nil {
-			log.Fatal(err)
-		}
-	}()
-}
-
-type cpuSink struct {
+// CPUSink is a CPU port client for a forwarding context.
+type CPUSink struct {
```

Comment:
nit: I remember a style guide saying this should be `Sink`, so that importers will see `cpusink.Sink`. `sink.CPUSink` suggests that there are other types of sinks (maybe there are, but the directory structure doesn't indicate it.

Reply:
renamed to sink